staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1/types.go

```diff
@@ -186,3 +186,346 @@ type WebhookMatchCondition struct {
 	// Documentation on CEL: https://kubernetes.io/docs/reference/using-api/cel/
 	Expression string `json:"expression"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// AuthenticationConfiguration provides versioned configuration for authentication.
```

Comment:
the order of the structs in staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1beta1/types.go and this file ( staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go ) is now different. so it gets harder to compare over time. Do we want to keep the order the same so a diff would be clearer?

Reply:
I can confirm that is is a cut-n-paste for v1beta1 for sure :)

Reply:
yeah, if it's easy to add the types to the v1/types.go file so they match the relative order in the v1beta1 types.go file for diffing, that would be nice

Reply:
done!